students/briannagata/session03/mailroom/actions.py

```diff
@@ -0,0 +1,30 @@
+# Modules to process selections made in mailroom.py
+# Python 3.6
+
+
+def list_database(donors):
+    """ Print the list of donor names"""    
+    for name in donors.keys():
+        print(name)
+
+
+def send_thankyou(name, money, text):
+    """ Emulate a thank you email by emulating content to terminal. """
+    text = text.replace('[DONOR]', name)
+    text = text.replace('[DONATION]', '{:.2f}'.format(money))
+    print(text, end='\n')
+
```

Comment:
Clever way of making use of a thank you letter template. This is great, since it completely separates the letter formatting from the data, making testing easier. All you have to do is make sure you are sending the correct name and money in.

Reply:
Took liberty to find a way to avoid repeating myself:
```
def send_thankyou(name, money, text):
    """ Emulate a thank you email by emulating content to terminal. """
    replacements = {'[DONOR]': name,
                    '[DONATION]': '{:.2f}'.format(money)}

    for key, value in replacements.items():
        text = text.replace(key, value)

    print(text, end='\n')
```